### PR TITLE
Update bloop

### PR DESCRIPTION
**Pull request by talum**

### CODE REVIEW

File: bloop [hunk 1]
```diff
@@ -1,5 +1,7 @@
-bloop
 
-blooping from @tracylum
+blooping sdfsdf
+
```
Comment (talum) [inline bloop:3]: from the other side

Comment (talum) [inline bloop:3]: sdfsf

File: bloop [hunk 2]
```diff
@@ -1,5 +1,7 @@
-bloop
 
-blooping from @tracylum
+blooping sdfsdf
```
Comment (talum) [inline bloop:2]:
```suggestion
no bloop
```

Comment (talum) [reply bloop]: another comment

File: bloop
Comment (talum) [inline bloop:1]: new file level comment and thread

Comment (talum) [reply bloop:1]: reply

File: bloop [hunk 3]
```diff
@@ -1,5 +1,7 @@
-bloop
 
-blooping from @tracylum
+no bloop
+
+
```
Comment (talum) [inline bloop:4]: Another comment

File: bloop [hunk 4]
```diff
@@ -1,5 +1,7 @@
-bloop
 
-blooping from @tracylum
+no bloop
+
```
Comment (talum) [inline bloop:3]: another thread

File: bloop
Comment (talum) [inline bloop:1]: pending comment

**Review by talum — COMMENTED** [review]
le boeuf

File: bloop
Comment (talum) [inline bloop:1]: Another comment

Comment (talum) [reply bloop:1]: reply

File: bloop [hunk 5]
```diff
@@ -1,5 +1,7 @@
-bloop
 
-blooping from @tracylum
```
Comment (talum) [inline bloop:3]: add comment to single file view

File: bloop (hunk 3 above)
Comment (talum) [inline bloop:4]: range comment

File: bloop
Comment (talum) [inline bloop:1]: hi

File: bloop (hunk 4 above)
Comment (talum) [inline bloop:3]: L1 to R3

File: bloop [hunk 6]
```diff
@@ -1,5 +1,7 @@
...
+no bloop
```
Comment (talum) [inline bloop:2]: sdfsdf

File: bloop (hunk 3 above)
Comment (talum) [inline bloop:4]: multiline

Comment (talum) [inline bloop:4]: L3 to R4

File: bloop (hunk 6 above)
Comment (talum) [inline bloop:2]: L1 to R2

File: bloop (hunk 3 above)
Comment (talum) [inline bloop:4]: L3 to R4

Comment (talum) [inline bloop:4]: L3 to R4

Comment (talum) [inline bloop:4]: sdfsdf

Comment (talum) [inline bloop:4]: from action bar plus icon

Comment (talum) [inline bloop:4]: first line

Comment (talum) [inline bloop:4]: L3 to R4 first line

Comment (talum) [inline bloop:4]: L3 to R4 last line plus icon

File: bloop [hunk 7]
```diff
@@ -1,5 +1,7 @@
...
+
+
+l;aksjd;lfkajsd;f
```
Comment (talum) [inline bloop:5]: L3 to R5

File: bloop (hunk 6 above)
Comment (talum) [inline bloop:2]: L3 to R2?

File: bloop (hunk 7 above)
Comment (talum) [inline bloop:5]: test?

File: bloop (hunk 5 above)
Comment (talum) [inline bloop:3]: test

File: bloop (hunk 6 above)
Comment (talum) [inline bloop:2]: test

File: bloop (hunk 7 above)
Comment (talum) [inline bloop:5]: sdfsdf

File: bloop (hunk 5 above)
Comment (talum) [inline bloop:3]: test

File: bloop (hunk 4 above)
Comment (talum) [inline bloop:3]: testing

File: bloop [hunk 8]
```diff
@@ -1,5 +1,7 @@
-bloop
```
Comment (talum) [inline bloop:1]: test

File: bloop (hunk 6 above)
Comment (talum) [inline bloop:2]: range

File: bloop (hunk 4 above)
Comment (talum) [inline bloop:3]:
```suggestion
no bloop
```

File: bloop
Comment (talum) [inline bloop:1]: file level

File: bloop (hunk 8 above)
Comment (talum) [inline bloop:1]: from single file view

File: bloop
Comment (talum) [reply bloop:1]: single file

Comment (talum) [inline bloop:1]: new comment single file

File: bloop (hunk 4 above)
Comment (talum) [inline bloop:3]:
```suggestion
no bloop
```

action bar

File: bloop
Comment (talum) [inline bloop:1]: my pending comment from files

Comment (talum) [inline bloop:1]: pending

Comment (talum) [inline bloop:1]: another pending comment from files

Comment (talum) [inline bloop:1]: file level comment on bloop from files

File: bloop (hunk 4 above)
Comment (talum) [inline bloop:3]:
```suggestion
```

File: bloop (hunk 7 above)
[the remark this answers is not in the document]
Comment (talum) [reply bloop:5]:
```suggestion
change
```

**Review by tracylum — CHANGES_REQUESTED** [review]
stahp

**Pull request merged**